app/views/shared/_messages.html.slim

```diff
@@ -2,4 +2,4 @@
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
       div class="alert alert-#{name}" role='alert'
-        = msg
+        = sanitize(msg)
```

Comment:
`safe_join` produces this result:
![screen shot 2017-02-03 at 11 50 52 am](https://cloud.githubusercontent.com/assets/1421848/22600094/11bae24a-ea07-11e6-8fd1-f5c832cdcd3e.png)